config/authorization/config.ex

```diff
@@ -40,13 +40,18 @@ defmodule Acl.UserGroups.Config do
 
   defp minister_roles do
     [
-      "<http://themis.vlaanderen.be/id/gebruikersrol/01ace9e0-f810-474e-b8e0-f578ff1e230d>" # minister
+      "<http://themis.vlaanderen.be/id/gebruikersrol/01ace9e0-f810-474e-b8e0-f578ff1e230d>" # minister and kabinetchef
```

Comment:
Het is mogelijk verwarrend/een gevoeligheid dat de kabinetschef in het gebruikersoverzicht/rechtsboven wanneer ingelogd gecategoriseerd wordt als "minister". Geen blocker. Misschien voor later.